### ISSUE TITLE
build fails with poppler-0.71.0

### ISSUE THREAD

**Issue by Polynomial-C** [OpenPrinting/cups-filters#69]
```
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.       -I/usr/include/libpng16 -I/usr/include/poppler  -I./cupsfilters/ -march=native -mtune=native -O2 -pipe -std=c++11 -Wall  -D_GNU_SOURCE -c -o pdftoraster-pdftoraster.o `test -f 'filter/pdftoraster.cxx' || echo './'`filter/pdftoraster.cxx
filter/pdftoraster.cxx: In function ‘void outPage(PDFDoc*, Catalog*, int, SplashOutputDev*, cups_raster_t*)’:
filter/pdftoraster.cxx:1689:45: error: invalid conversion from ‘const PDFRectangle*’ to ‘PDFRectangle*’ [-fpermissive]
   PDFRectangle *mediaBox = page->getMediaBox();
                            ~~~~~~~~~~~~~~~~~^~
filter/pdftoraster.cxx:1817:6: error: ‘gTrue’ was not declared in this scope
      gTrue,gTrue,gTrue);
      ^~~~~
filter/pdftoraster.cxx:1817:6: note: suggested alternative: ‘gfree’
      gTrue,gTrue,gTrue);
      ^~~~~
      gfree
filter/pdftoraster.cxx: In function ‘int main(int, char**)’:
filter/pdftoraster.cxx:1970:20: error: invalid conversion from ‘void (*)(void*, ErrorCategory, Goffset, char*)’ {aka ‘void (*)(void*, ErrorCategory, long long int, char*)’} to ‘void (*)(void*, ErrorCategory, Goffset, const char*)’ {aka ‘
void (*)(void*, ErrorCategory, long long int, const char*)’} [-fpermissive]
   setErrorCallback(::myErrorFun,NULL);
                    ^~
In file included from /usr/include/poppler/Object.h:45,
                 from filter/pdftoraster.cxx:44:
/usr/include/poppler/Error.h:50:37: note:   initializing argument 1 of ‘void setErrorCallback(void (*)(void*, ErrorCategory, Goffset, const char*), void*)’
 extern void setErrorCallback(void (*cbk)(void *data, ErrorCategory category,
                              ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       Goffset pos, const char *msg),
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
filter/pdftoraster.cxx:2127:5: error: ‘gFalse’ was not declared in this scope
     gFalse,paperColor,gTrue
     ^~~~~~
filter/pdftoraster.cxx:2127:5: note: suggested alternative: ‘pause’
     gFalse,paperColor,gTrue
     ^~~~~~
     pause
filter/pdftoraster.cxx:2127:23: error: ‘gTrue’ was not declared in this scope
     gFalse,paperColor,gTrue
                       ^~~~~
filter/pdftoraster.cxx:2127:23: note: suggested alternative: ‘gfree’
     gFalse,paperColor,gTrue
                       ^~~~~
                       gfree
make[1]: *** [Makefile:4366: pdftoraster-pdftoraster.o] Error 1
make[1]: Leaving directory '/var/tmp/portage/net-print/cups-filters-9999/work/cups-filters-9999'
make: *** [Makefile:2076: all] Error 2
```
At least the `gTrue` and `gFalse` issues are caused by the following commit from poppler upstream:
[Replace GBool, gTrue, and gFalse by bool, true, false, resp.](https://cgit.freedesktop.org/poppler/poppler/commit/?id=163420b48bdddf9084208b3cadf04dafad52d40a)

**Comment by Polynomial-C** [comment]
This is still not fixed:
```
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.       -I/usr/include/libpng16 -I/usr/include/poppler  -I./cupsfilters/ -march=native -mtune=native -O2 -pipe -std=c++11 -Wall  -D_GNU_SOURCE -c -o pdftoraster-pdftoraster.o `test -f 'filter/pdftoraster.cxx' || echo './'`filter/pdftoraster.cxx
filter/pdftoraster.cxx: In function ‘void outPage(PDFDoc*, Catalog*, int, SplashOutputDev*, cups_raster_t*)’:
filter/pdftoraster.cxx:1821:6: error: ‘gTrue’ was not declared in this scope
      gTrue,gTrue,gTrue);
      ^~~~~
filter/pdftoraster.cxx:1821:6: note: suggested alternative: ‘gfree’
      gTrue,gTrue,gTrue);
      ^~~~~
      gfree
filter/pdftoraster.cxx: In function ‘int main(int, char**)’:
filter/pdftoraster.cxx:2131:5: error: ‘gFalse’ was not declared in this scope
     gFalse,paperColor,gTrue
     ^~~~~~
filter/pdftoraster.cxx:2131:5: note: suggested alternative: ‘pause’
     gFalse,paperColor,gTrue
     ^~~~~~
     pause
filter/pdftoraster.cxx:2131:23: error: ‘gTrue’ was not declared in this scope
     gFalse,paperColor,gTrue
                       ^~~~~
filter/pdftoraster.cxx:2131:23: note: suggested alternative: ‘gfree’
     gFalse,paperColor,gTrue
                       ^~~~~
                       gfree
make[1]: *** [Makefile:4366: pdftoraster-pdftoraster.o] Error 1
make[1]: Leaving directory '/var/tmp/portage/net-print/cups-filters-9999/work/cups-filters-9999'
make: *** [Makefile:2076: all] Error 2
```

**Comment by tillkamppeter** [comment]
Fixed in commit 48625da3f.
